tests/test_address.py

```diff
@@ -6,8 +6,32 @@
 from cdp.address import Address
 from cdp.balance_map import BalanceMap
 from cdp.client.exceptions import ApiException
+from cdp.client.models.asset import Asset as AssetModel
+from cdp.client.models.balance import Balance as BalanceModel
 from cdp.errors import ApiError
 from cdp.faucet_transaction import FaucetTransaction
+from cdp.historical_balance import HistoricalBalance
+from cdp.transaction import Transaction
+
+
+@pytest.fixture
+def address():
+    """Create and return a fixture for an Address."""
+    return Address(
+        network_id="base-sepolia", address_id="0x1234567890123456789012345678901234567890"
+    )
+
+
+@pytest.fixture
+def asset_model():
+    """Create and return a fixture for an AssetModel."""
+    return AssetModel(network_id="base-sepolia", asset_id="eth", decimals=18)
+
+
+@pytest.fixture
+def balance_model(asset_model):
+    """Create and return a fixture for a BalanceModel."""
+    return BalanceModel(amount="1000000000000000000", asset=asset_model)
```

Comment:
Can we please use fixture factories in `./tests/factories`

Reply:
Ops, sorry this is probably merge issue, let me remove those. thanks!